### PR TITLE
[gntai] removed blogger template

### LINKED ISSUE #2288: http://www.gntai.net/
**Name of the website**
GNTAI

**Website urls (examples below)**
- Site: http://www.gntai.net/
- Manga List: http://www.gntai.net/?s=
- Manga example: http://www.gntai.net/?s=Saimin+Seishidou
- Chapter online viewer example: http://www.gntai.net/mangas-hentai/saimin-seishidou-1-obata-yui-to-tachibana-daiki-no-baai/

**Languages**
Spanish

**Additional details**
Que se pueda acceder a la lista y se pueda actulizar ademas de tambien solo poner el link del doujin y que salga directo

### PR DESCRIPTION
**Pull request by 27136773**
References: dazedcat19/FMD2#204

### CODE REVIEW

File: src/web/mjs/connectors/Gntai.mjs [hunk 1]
```diff
@@ -1,57 +1,51 @@
...
-                    language: ''
-                } ];
-                callback( null, chapterList );
-            } )
-            .catch( error => {
-                console.error( error, manga );
-                callback( error, undefined );
-            } );
+    async _getChapters(manga) {
+        return [ Object.assign({ language: '' }, manga) ];
```
Comment (Sheepux) [inline src/web/mjs/connectors/Gntai.mjs:44]: I don't get that part. Are you just returning the manga with a language flag ? How does the chapters get retrieved ?

Comment (27136773) [reply src/web/mjs/connectors/Gntai.mjs:44]: Hentai websites are usually one-shots without chapters, the manga is basically the one shot chapter.
It creates a new object which acts as chapter containing a `language` property and then assigned the `id` and `title` property from the manga to it.

**Review by Sheepux — APPROVED** [review]

File: src/web/mjs/connectors/Gntai.mjs (hunk 1 above)
Comment (27136773) [reply src/web/mjs/connectors/Gntai.mjs:44]: Samples for clarification of `Object.assign`:

```javascript
let source = { b: 99, x: 0 };
let target = { a: 1, b: 2 };
Object.assign(target, source);
console.log(source, target);
```

```javascript
let manga = { id: '/comic/my-little-pony', title: 'My Big Pony' };
let chapter = { language: 'pony' };
Object.assign(chapter, manga);
console.log(chapter);
```

Comment (Sheepux) [reply src/web/mjs/connectors/Gntai.mjs:44]: Object assign was ok, it was just the fact that the manga was returned that disturbed me. The previous statement clariify everything :)

**Pull request merged**